eng/precommit_local_build.py

```diff
@@ -73,15 +73,17 @@ def get_artifacts_from_pom(pom_path: str, build_artifacts: list, debug: bool):
 
 def main():
     parser = argparse.ArgumentParser(description='Runs compilation, testing, and linting for the passed artifacts.')
-    parser.add_argument('--artifacts', '--a', type=str, default=None, help='Comma separated list of groupId:artifactId identifiers')
-    parser.add_argument('--poms', '--p', type=str, default=None, help='Comma separated list of POM paths')
-    parser.add_argument('--skip-tests', '--st', action='store_true', help='Skips running tests')
-    parser.add_argument('--skip-javadocs', '--sj', action='store_true', help='Skips javadoc generation')
-    parser.add_argument('--skip-checkstyle', '--sc', action='store_true', help='Skips checkstyle linting')
-    parser.add_argument('--skip-spotbugs', '--ss', action='store_true', help='Skips spotbugs linting')
-    parser.add_argument('--skip-revapi', '--sr', action='store_true', help='Skips revapi linting')
-    parser.add_argument('--command-only', '--co', action='store_true', help='Indicates that only the command should be generated and not ran')
-    parser.add_argument('--debug', '--d', action='store_true', help='Generates command with verbose logging')
+    parser.add_argument('--artifacts', '-a', type=str, default=None, help='Comma separated list of groupId:artifactId identifiers')
+    parser.add_argument('--poms', '-p', type=str, default=None, help='Comma separated list of POM paths')
+    parser.add_argument('--skip-tests', '-st', action='store_true', help='Skips running tests')
+    parser.add_argument('--skip-javadocs', '-sj', action='store_true', help='Skips javadoc generation')
+    parser.add_argument('--skip-checkstyle', '-sc', action='store_true', help='Skips checkstyle linting')
+    parser.add_argument('--skip-spotbugs', '-ss', action='store_true', help='Skips spotbugs linting')
+    parser.add_argument('--skip-revapi', '-sr', action='store_true', help='Skips revapi linting')
+    parser.add_argument('--skip-readme', '-smd', action='store_true', help='Skips README validation')
```

Comment:
Do we have an option to skip gpg check?

Reply:
Not yet, that could be added in the future